src/libraries/System.Net.Requests/src/System/Net/FileWebRequest.cs

```diff
@@ -9,6 +9,8 @@
 
 namespace System.Net
 {
+    // NOTE: While this class is not explicitly marked as obsolete,
+    // it effectively is by virtue of WebRequest.Create being obsolete.
```

Comment:
Why haven't we explicitly marked it?

Reply:
I believe it was to reduce the noise for anyone still using it without suppressing it at the project level.
They must suppress the warning at least once when creating the request, but not everywhere else they might have plumbed the types through.